### ISSUE TITLE
No docs on how to provide the server port as parameter

### ISSUE THREAD

**Issue by xanathon** [digital-dream-labs/vector-web-setup#26]
In the latest changelog is the information that you can now specify a port when starting the app.

However, neither in the changelog nor in the main readme is any information whatsoever on how to do this.

I know that this can be found in an issue somewhere, but in my opinion this needs to be in the documentation where it can easily be found.

**Comment by grant-olson** [comment]
We will add something to the documentation but you can also use the `--help` flag to get a global help page or get the options for any given command such as `vector-web-setup serve --help`:

```
Usage: vector-web-setup serve [options]

Serve the vector websetup

Options:
  -p, --port <number>       port to serve the setup on (default: 8000)
  -ip, --ip-address <addr>  address to serve the setup on (default: "0.0.0.0")
  -h, --help                display help for command

```

**Comment by grant-olson** [comment]
This was fixed by pr #29
